app/main.py

```diff
@@ -2,0 +1,12 @@
+def copy_file(command: str) -> None:
+    parts = command.split()
+
+    if len(parts) != 3 or parts[0] != "cp":
+        return
+
+    src_file, dst_file = parts[1], parts[2]
+    if src_file == dst_file:
```

Comment:
The condition to check if the source and destination files are the same is correct. This aligns with the task requirement: 'It must do nothing in case the user is trying to copy file to file with the same name.'